frontend/config/routes.rb

```diff
@@ -2,10 +2,11 @@
   scope '(:locale)', locale: /#{Spree.available_locales.join('|')}/, defaults: { locale: nil } do
     root to: 'home#index'
 
-    resources :products, only: [:index, :show]
-    get '/products/:id/related', to: 'products#related'
+    resources :products, only: [:index, :show], path: "/#{Spree::Config[:storefront_product_path]}"
```

Comment:
@MatthewKennedy how will that work for index? will it automatically translate it to plural form? Maybe we can just create a separate singular path for product show action?

Reply:
Ah, gotcha! However, maybe let's leave the index alone and only apply the config for show action as the name of the config is plural and most people will think this will apply only to show action. The index action is only used for search results anyway

Reply:
@damianlegawiec I think the config name I have used is what is throwing us off here, as the actual paths are pluralised as you would expect both the index and the show action:

`preference :storefront_product_path, :string, default: 'products'` resolves to: `products` as you would expect for both the `show` and `index` actions.

I think I should pluralised the config names so it makes sense in this context:

From: `preference :storefront_product_path, :string, default: 'products'`
To: `preference :storefront_products_path, :string, default: 'products'`